.gitignore

```diff
@@ -8,3 +8,4 @@ vendor/
 composer.lock
 /phpunit.xml
 /.phpcs-cache
+.idea
```

Comment:
This should be in your [global `.gitignore`](https://gist.github.com/subfuzion/db7f57fff2fb6998a16c), and not in the project